test/e2e/go.mod

```diff
@@ -1,29 +1,31 @@
 module github.com/nutanix-cloud-native/cloud-provider-nutanix/test/e2e
 
-go 1.23
+go 1.23.0
 
 toolchain go1.23.7
 
 require (
-	github.com/onsi/ginkgo/v2 v2.20.2
-	github.com/onsi/gomega v1.34.2
-	k8s.io/api v0.31.1
-	k8s.io/apimachinery v0.31.1
-	sigs.k8s.io/cluster-api v1.8.3
-	sigs.k8s.io/cluster-api/test v1.8.3
-	sigs.k8s.io/controller-runtime v0.19.0
+	github.com/onsi/ginkgo/v2 v2.22.2
+	github.com/onsi/gomega v1.36.2
+	k8s.io/api v0.32.2
+	k8s.io/apimachinery v0.32.2
+	sigs.k8s.io/cluster-api v1.9.0-rc.0.0.20250228131023-67245f572cbe
```

Comment:
Why not v1.9.5?

Reply:
https://github.com/nutanix-cloud-native/cloud-provider-nutanix/pull/201#discussion_r1975885463